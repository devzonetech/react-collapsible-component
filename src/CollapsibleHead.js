import React from 'react'

export class CollapsibleHead extends React.Component {
    
    render() {

        const { className } = this.props;
        let addClassName = className ? className : '';

        return (<button data-show-content-above-button={this.props.showContentAboveButton}
        className={'accordion-head ' + (this.props.isExpanded ? 'active-accordion ' : '') + addClassName}>
            {this.props.children}
        </button>)
    }
}