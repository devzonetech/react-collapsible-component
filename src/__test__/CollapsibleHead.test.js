import React from 'react'
import renderer from 'react-test-renderer'
import CollapsibleHead from '../CollapsibleHead'

describe('CollapsibleHead', () => {
    test('snapshot renders', () => {
        const component = renderer.create(<CollapsibleHead />)
        let tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})