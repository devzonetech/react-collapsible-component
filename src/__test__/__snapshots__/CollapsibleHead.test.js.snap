// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollapsibleHead snapshot renders 1`] = `
<button
  className="accordion-head "
/>
`;
